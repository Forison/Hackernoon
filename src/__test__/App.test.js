import React from 'react';
import renderer from 'react-test-renderer';


it('renders content with H1 tag with text call related', () => {
  const tree = renderer
    .create(<h1>Facebook</h1>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

