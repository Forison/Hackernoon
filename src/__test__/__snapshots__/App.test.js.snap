// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders content with H1 tag with text call related 1`] = `
<h1>
  Facebook
</h1>
`;
